packages/components/src/core/Autocomplete/index.tsx

```diff
@@ -193,6 +207,16 @@ const Autocomplete = <
     />
   );
 
+  function clearInput() {
+    setInputValue("");
+    if (onInputChange)
```

Comment:
Because we are manually firing this event, we must build a onChange event to transmit the updated value to onChange listeners.

Reply:
let's add this context comment in the code too please 🙏